src/lib/wizards/functions/steps/manualDetails.svelte

```diff
@@ -53,17 +53,19 @@
             {options}
             required />
 
-        <InputSelect
-            label="CPU and memory"
-            id="specification"
-            placeholder="Select specification"
-            required
-            disabled={specificationOptions.length < 1}
-            options={specificationOptions}
-            popover={isCloud && $organization?.billingPlan === BillingPlan.FREE
-                ? SpecificationsTooltip
-                : null}
-            bind:value={$createFunction.specification} />
+        {#if isCloud}
+            <InputSelect
+                label="CPU and memory"
+                id="specification"
+                placeholder="Select specification"
+                required
+                disabled={specificationOptions.length < 1}
+                options={specificationOptions}
+                popover={isCloud && $organization?.billingPlan === BillingPlan.FREE
```

Comment:
already `isCloud` here no?

---
Also lets move from doing a plan check to checking if the specification list only has one item [Free only has one]. @lohanidamodar makes sense?